hydpy/models/whmod/whmod_script.py

```diff
@@ -382,56 +605,46 @@ def run_whmod(basedir: str, write_output: str) -> None:
     for element in hp.elements:
         element.model.parameters.update()
 
-    # define two loggers, one for the actual groundwater recharge, one for the delayed
-    # groundwater recharge (verz)
-    hp.loggers["logger_akt"] = logtools.Logger(simulation_start, simulation_end)
-    hp.loggers["logger_verz"] = logtools.Logger(simulation_start, simulation_end)
+    # Define Loggers according to OUTPUTCONFIG
+    loggers = []
+    for file in outputconfig:
+        loggers.append(read_outputconfig(basedir=basedir, outputconfigfile=file))
+    hydpy.pub.selections += hydpy.Selection(
+        name="complete", nodes=hp.nodes, elements=hp.elements
+    )
+    whm_elements = (
+        hydpy.pub.selections["complete"].search_modeltypes("whmod_pet").elements
+    )
 
-    # same for the month logger
-    if "txt" in outputmode or "rch" in outputmode:
-        hp.loggers["month_logger_akt"] = whmod_model.WHModMonthLogger()
-        hp.loggers["month_logger_verz"] = whmod_model.WHModMonthLogger()
+    seriesdir = os.path.join(outputdir, "series")
+    hydpy.pub.sequencemanager.currentdir = seriesdir
+    hydpy.pub.sequencemanager.filetype = "nc"
 
-    # add the fluxes to the respective logger
-    for element in hp.elements:
-        if element.name.startswith("WHMod"):
-            hp.loggers["logger_akt"].add_sequence(
-                element.model.sequences.fluxes.aktgrundwasserneubildung
-            )
-            hp.loggers["month_logger_akt"].add_sequence(
-                element.model.sequences.fluxes.aktgrundwasserneubildung
-            )
-            hp.loggers["logger_verz"].add_sequence(
-                element.model.sequences.fluxes.verzgrundwasserneubildung
-            )
-            hp.loggers["month_logger_verz"].add_sequence(
-                element.model.sequences.fluxes.verzgrundwasserneubildung
-            )
+    for element in whm_elements:
+        for logger in loggers:
+            for seq in logger["sequence"]:
+                sequence = getattr(element.model.sequences.fluxes, seq.lower())
+                if not sequence.diskflag_writing:
+                    sequence.prepare_series(allocate_ram=False, write_jit=True)
 
     hp.simulate()
 
     hydpy.pub.sequencemanager.overwrite = True
     hydpy.pub.sequencemanager.currentdir = outputdir
 
-    ncol = df_knoteneigenschaften["col"].max()
-    nrow = df_knoteneigenschaften["row"].max()
-    xllcorner = df_knoteneigenschaften["x"].min()
-    yllcorner = df_knoteneigenschaften["y"].min()
+    if write_output:
+        commandtools.print_textandtime(f"Write Output in {outputdir}")
```

Comment:
Some doctests need to be updated. This seems to be the only cause why all AppVeyor jobs currently fail.